data/boundary-composition/ascii-data/test/box_2d_left.0.csv

```diff
@@ -0,0 +1,7 @@
+# Test data for ascii data boundary composition.
+# Only next line is parsed in format: [nx] because of keyword "POINTS:"
+# POINTS: 3
+# Columns: x composition_1
```

Comment:
I like this file format much better. Out of curiosity, why do the files have suffix `.csv` (comma-separated values) when in fact the values are separated by spaces? ;-)